public func foo(
    arg1: Int32
) -> Bool { fatalError() }